### ISSUE TITLE
packages out of date

### ISSUE THREAD

**Issue by bion** [re5et/.emacs.d-for-my-friends#2]
wow

**Comment by re5et** [comment]
Yeah I would say very out of date.  Haven't been doing anything with this because I don't think anyone uses it.

At this point, it doesn't make sense to commit packages as more recent versions of emacs have decent enough to use package management.

**Comment by bion** [comment]
aaaaah aha. i was gonna steer a couple people to it.

**Comment by re5et** [comment]
wanna work on it :p ?
